0007/README.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 0007
+title: PlaywithTink and the user persona
+status: discussion
+authors: Gianluca Arbezzano <gianarb92@gmail.com>
+---
+
+## Summary
+
+The setup workflows and tutorial we have today in my opinion are for
+contributors. For developers who want to code for Tinkerbell.
+
+Because the process checkouts the `tinkerbell/tink` repository, and it runs the latest version of
+dependencies like: osie, tink-server, tink-worker, boots and so on. This is a
+big problem today because all those components are under development, and we
+can't guarantee that what it is in master works. Even less in integration with
+the other components. The entropy is too high.
+
+## Goals and non-Goals
+
+Goals:
+
+1. Figure out a way to serve a pinned "version" of Tinkerbell that does not
+   use the current master version but something that is tested.
+2. Avoid compiling or building code or docker images in favor of using already
+   available solutions.
+
+Non-Goals:
+
+1. Tag a release for project, we are not ready for that and there is proposal
+   0002 under discussion about that.
+2. Getting to something that will work forever, we need something that works
+   NOW.
+
+## Content
+
+My proposal is a shortcut to get to something working quickly. I am not here to
+evaluate the current experience. As a contributors and developers we set that up
+in that way, and we are developing Tinkerbell, so it works.
+
+I don't like to propose a solution for our user that is too far away from that
+because:
+
+1. It will help contributors to become user and even more important vice-versa.
+2. We will be able to reuse some code (vagrant e2e tests) to validate this
+   solution
+3. It is validated by the community already.
+
+We can have a new repository called: `tinkerbell/playwithtink` it will contain
+a setup.sh copy/paste or equivalent script and a `docker-compose` file like the
+one we serve in `tinkerbell/tink` but with pinned docker images.
+
+We can replace or write setup guides as similar as the one we already have, but
+they won't checkout the `tinkerbell/tink` repository, but they will `wget` the
+static files served by `tinkerbell/playwithtink` (as a release if we want extra
+safety) in this way the version in use will stay the same no matter what happens
```

Comment:
The `version` here is very important. At present, the docker-compose uses `latest` tag which for all the right reasons is not a good practice. 

We will have to be clear about `tinkerbell/playwithtink` in our documentation (at [tinkerbell.org](tinkerbell.org)) so that people are not confused which setup instructions to follow.

Reply:
I tried to explain as part of the RFC that we can decide if we want to replace the current setup guide with playwithtink. Is it not clear enough?! I can elaborate on it more.

Reply:
Yeah, it's pretty clear. 👍🏼 
I only wanted to understand where we want to document the developer steps for tinkerbell setup. And I think we can have them with Contribution guideline under the `tink` repository.